cpp/modmesh/toggle/pymod/wrap_Toggle.cpp

```diff
@@ -228,25 +426,63 @@ WrapToggle::WrapToggle(pybind11::module & mod, char const * pyname, char const *
     (*this)
         .def("clone", &wrapped_type::clone, py::return_value_policy::take_ownership)
         .def("report", &report)
+        .def("to_python", detail::Toggle2Python::from_toggle, py::arg("type") = "")
         //
         ;
 
-    // Dynamic properties.  Number of the properties can be freely changed
-    // during runtime.
+    // Accssors to dynamic properties.  Number of the properties can be freely
+    // changed during runtime.
     (*this)
         .def(
-            "__getattr__",
+            "get_value",
             [](wrapped_type & self, std::string const & key)
             {
                 HierarchicalToggleAccess access(self.dynamic());
                 return WrapHierarchicalToggleAccess::getattr(access, key);
             })
+        .def(
+            "__getattr__",
+            [](wrapped_type & self, std::string const & key)
+            {
+                HierarchicalToggleAccess access(self.dynamic());
```

Comment:
This is about the longest code I can tolerate in a lambda.  Later when we have more sophisticated toggle structure this will need to be refactored.